packages/dd-trace/test/appsec/iast/overhead-controller.spec.js

```diff
@@ -0,0 +1,112 @@
+const overheadController = require('../../../src/appsec/iast/overhead-controller')
+const DatadogSpanContext = require('../../../src/opentracing/span_context')
+
+describe('Overhead controller', () => {
+  const oceContextKey = overheadController.OVERHEAD_CONTROLLER_CONTEXT_KEY
+
+  beforeEach(() => {
+    overheadController._resetGlobalContext()
+  })
+
+  describe('Initialize OCE context', () => {
+    describe('Request context', () => {
+      it('should populate request context', () => {
+        const iastContext = {}
+        overheadController.initializeRequestContext(iastContext)
+        expect(iastContext).to.have.nested.property(overheadController.OVERHEAD_CONTROLLER_CONTEXT_KEY)
+      })
+    })
+
+    describe('Global context', () => {
+      before(() => {
+        sinon.spy(global, 'setInterval')
+        sinon.spy(global, 'clearInterval')
+      })
+
+      afterEach(() => {
+        sinon.restore()
+      })
+
+      it('should start and stop global context refresh interval just once', () => {
+        overheadController.startGlobalContextResetInterval()
+        overheadController.startGlobalContextResetInterval()
+        overheadController.stopGlobalContextResetInterval()
+        overheadController.stopGlobalContextResetInterval()
+        expect(setInterval).to.have.been.calledOnce
+        expect(clearInterval).to.have.been.calledOnce
+      })
+    })
+  })
+
+  describe('Analyze request', () => {
+    it('should allow requests which span id is divisible by 3', () => {
+      const rootSpan = {
+        context: sinon.stub().returns(new DatadogSpanContext({
+          spanId: 3
+        }))
+      }
+
+      const reserved = overheadController.acquireRequest(rootSpan)
+      expect(reserved).to.be.true
+    })
+  })
+
+  it('should not allow requests which span id is not divisible by 3', () => {
```

Comment:
Isn't this test part of `Analyze request`? It is out of the `describe('Analyze request'`